js/api-augment/definitions/msa.ts

```diff
@@ -47,14 +36,24 @@ export default {
       ],
       type: "Option<Vec<SchemaGrantResponse>>",
     },
+    getKeysByMsaId: {
```

Comment:
If I run `make start-interval-with-offchain`, should I be able to access this via polkadot UI->Developer->RPC calls->msa?

Reply:
You can access that RPC directly by http call without having `with-offchain` but it would be empty or error out. My impression is that to be able to see the rpc from polkadot UI I think we need to submit PR on polkadot UI once in a while to be able to access them via UI.

Reply:
Correct--extrinsic changes are visible immediately as they rely on on-chain information, but knowledge of RPCs is built into the polkadot-js JS package, so you have to wait for a Frequency release and _then_ the next PolkadotJS release after that to access new RPCs in the UI.